doc/databinding/@prop.md

```diff
@@ -2,21 +2,32 @@
 ---
 # @prop
 
-An alias for [`@property`](./@property.md), except that it has "smarter" default options and a different shorthand parameter. It's generally recommended over `@property` for custom component development, but its default behavior may be unexpected for newcomers.
+An alias for [`@property`](./@property.md), except that it has "smarter" default options and a different shorthand parameter. It's generally recommended over `@property` for custom component development, but its default behavior may be unexpected for newcomers:
+
+* Mixed types ("unions" in TypeScript) are not supported.
```

Comment:
This is not specific to the differences between `@prop` and `@property`? Or do I miss something?